templates/search/blocks/search_results__selected-filters.html

```diff
@@ -6,7 +6,7 @@ <h2 class="sr-only">Selected filters</h2>
     {% for field_name, selected_options in selected_filters.items %}
         {% for value, label in selected_options %}
             {# IMPORTANT: Never apply '|safe' to value or label here, as validation is not applied to most of these fields #}
-            <li class="search-results__selected-filters-item">
+            <li class="search-results__selected-filters-item" data-filter data-filter-value="{% if field_name == 'opening_start_date' %}{{ value|date:"o-m-d" }}{% elif field_name == 'opening_end_date' %}{{ value|date:"o-m-d" }}{% else %}{{ label }}{% endif %}" data-filter-name="{{ field_name }}" data-search-bucket="{{ buckets.current.label }}">
```

Comment:
Thanks @janehughes . It looks good. If we have more date filter fields, it looks like a candidate for refactor.

Reply:
Yep, agreed.